scripts/2.TCGA-process.py

```diff
@@ -59,11 +60,118 @@
 snp_mutation_df.effect.value_counts().reset_index()
 
 
+# ### Convert mutation gene symbol labels to Entrez IDs  
+
+# Goal: Relabel the mutation data frame with Entrez IDs instead of gene names, by mapping a combination of chromosome and gene symbol to Entrez ID. The NCBI file downloaded and read in the next cell contains the Entrez ID - gene symbol pairs we will use to do so.
+
+# In[7]:
+
+base_url = 'ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/'
+map_name = 'Homo_sapiens.gene_info.gz'
+map_url = base_url + map_name
+path = os.path.join('download', map_name)
+urlretrieve(map_url, path)
+
+path = os.path.join('download', 'Homo_sapiens.gene_info.gz')
+map_names = ['tax_id', 'GeneID', 'Symbol', 'LocusTag', 'Synonyms', 'dbXrefs',
+             'chromosome', 'map_location', 'description', 'type_of_gene',
+             'Symbol_from_nomenclature_authority', 'Full_name_from_nomenclature_authority',
+             'Nomenclature_status', 'Other_designations', 'Modification_date']
```

Comment:
Can you use the column names from the file and then just remove the `#` from `#tax_id`? Explicit naming could break if NCBI updates their file.